Library/Homebrew/cmd/info.rb

```diff
@@ -161,4 +169,11 @@ def decorate_dependencies(dependencies)
     end
     deps_status * ", "
   end
+
+  def decorate_requirements(requirements)
+    req_status = requirements.collect do |req|
+      req.satisfied? ? pretty_installed(req.display_s) : pretty_uninstalled(req.display_s)
```

Comment:
As over 80 characters I'd just make this an `if/else` and pull the `req.display_s` into a variable.